services/server/src/server/server.ts

```diff
@@ -70,13 +73,23 @@ export class Server {
     });
 
     logger.info("Starting server with config", {
-      config: JSON.stringify(config, null, 2),
+      options: JSON.stringify(options, null, 2),
+      verificationServiceOptions: JSON.stringify(verificationServiceOptions, null, 2),
+      storageServiceOptions: JSON.stringify(storageServiceOptions, null, 2),
     });
 
-    this.port = port;
+    this.port = options.port;
     logger.info("Server port set", { port: this.port });
     this.app = express();
 
+    this.chainRepository = new ChainRepository(options.chains);
+
+    this.app.set('chainRepository', this.chainRepository);
+    this.app.set('solc', options.solc);
+    this.app.set('verifyDeprecated', options.verifyDeprecated);
+    this.app.set('sessionOptions', options.sessionOptions);
```

Comment:
We don't use this anywhere
```suggestion
```

Reply:
Fixed in [ac173f1](https://github.com/ethereum/sourcify/pull/1637/commits/ac173f1bb615fd73c7982d75715860eb2b841529)